AstaZero/CMakeLists.txt

```diff
@@ -12,13 +12,16 @@ set (ROADMANAGER_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../EnvironmentSimulato
 add_subdirectory(${COMMON_MINI_INCLUDE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/build/commonMini)
 add_subdirectory(${ROADMANAGER_INCLUDE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/build/roadManager)
 
+add_executable(${TARGET} 
+  src/CalculateReferenceLine.cpp 
+  src/main.cpp
+  src/OverlappingReferenceLines.cpp
```

Comment:
This src file is not included in the PR?

Reply:
my mistake, probably in the other pr...

Reply:
Now it is fixed by adding the other PR in to this one